src/desktop/apps/artist/maybe_new_artist_page.ts

```diff
@@ -0,0 +1,11 @@
+// TODO: Read from ENV.
+const NEW_ARTIST_PAGE_THRESHOLD = 3
+
+module.exports = (_req, res, next) => {
+  console.log(res.locals.sd.NEW_ARTIST_PAGE)
+  if (parseInt(res.locals.sd.NEW_ARTIST_PAGE, 10) < NEW_ARTIST_PAGE_THRESHOLD) {
+    return next('route')
```

Comment:
This nexts out of the artist app, and so you'll get artist2 (served under /artist)